### PR TITLE
Add a CONTRIBUTING.md doc

### PR DESCRIPTION
**Pull request by lucapette**
As discussed in #28

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Setup your machine
+
+`releaser` is written in [Go](https://golang.org/).
+
+Prerequisites are:
+
+* Build:
+  * `make`
+  * [Go 1.6+](http://golang.org/doc/install)
```
Comment (lucapette) [inline CONTRIBUTING.md]: Not sure we are using already 1.7+ features (I ran into https://blog.golang.org/subtests recently and using them already everywhere..). I can update accordingly

Comment (caarlos0) [inline CONTRIBUTING.md]: Sure, let's make this 1.7+ 👍

**Comment by caarlos0** [comment]
LGTM 🍻

**Comment by lucapette** [comment]
@caarlos0 nice!!!

**Pull request merged**